apps/mobile-wallet/src/components/layout/BottomModal.tsx

```diff
@@ -267,11 +279,21 @@ const Handle = styled(Animated.View)`
   margin-top: -15px;
 `
 
+const Title = styled(AppText)`
+  position: absolute;
+  right: 0;
+  left: 0;
+  text-align: center;
+`
```

Comment:
With absolute positioning, if the title of a bottom modal is long (can be caused by translations of different screen resolutions) it will overflow on top of the "X" button

<img width="300" alt="image" src="https://github.com/user-attachments/assets/acfef568-a3b5-47a2-aac3-a9ce0c3b4605">

Reply:
The way I would address this would be to not use absolute positioning and add an invisible flexbox item on the left that has the same size as the "X" button on the right. Then, the title can take all available space and still be in the center.